sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/MetadataTests.java

```diff
@@ -124,6 +125,30 @@ CREATE VIEW V WITH (
                 }""", str);
     }
 
+    @Test
+    public void stripConnectors() throws IOException, SQLException {
+        // Test that the connectors property is stripped from the generated Rust
+        String sql = """
+               CREATE TABLE T (
+                  COL1 INT
+               ) WITH (
+                  'connectors' = '[{
+                    "name": "kafka",
+                    "url": "localhost"
+                  }]'
+               );
+               CREATE VIEW V WITH (
+                  'connectors' = '[{
+                     "name": "file_input",
+                     "path": "/tmp/x"
+                  }]'
+               ) AS SELECT * FROM T;""";
+        File file = createInputScript(sql);
+        CompilerMain.execute("-o", BaseSQLTests.testFilePath, file.getPath());
+        String rust = Utilities.readFile(Paths.get(BaseSQLTests.testFilePath));
+        Assert.assertFalse(rust.contains("connectors"));
+    }
```

Comment:
Could you test that two programs with a table and a view, but different connectors, generate the same Rust source?

Reply:
Modified the test to check this.
Had to do a few more tweaks.